presto-docs/src/main/sphinx/release/release-337.rst

```diff
@@ -0,0 +1,55 @@
+=========================
+Release 337 (XX Jun 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect results for inequality join involving ``NaN``. (:issue:`4120`)
+* Fix peak non-revocable memory metric in event listener. (:issue:`4096`
+* Improve performance of queries involving comparisons between ``DOUBLE`` or ``REAL`` values and integer types. (:issue:`3533`)
+* Add peak non-revocable memory metric to query stats. (:issue:`4096`)
+* Add support for variable-precision ``TIMESTAMP WITH TIME ZONE`` type (:issue:`3947`)
+* Add support for ``IN`` predicate with subqueries in outer join condition. (:issue:`4151`)
+* Add support for quantified comparisons (e.g., ``> ALL (...)``) in aggregation queries. (:issue:`4128`)
+* Add Druid connector. (:issue:`3522`)
+* Reduce worker graceful shutdown duration. (:issue:`4192`)
+
+Security Changes
+----------------
+
+* Disable user impersonation in default system security. (:issue:`4082`)
+
+Elasticsearch Connector Changes
+-------------------------------
+
+* Add support for password authentication. (:issue:`4165`)
+
+Hive Connector Changes
+----------------------
+
+* Fix reading CSV tables with ``separatorChar``, ``quoteChar`` or ``escapeChar`` table property
+  containing more than one character. For compatibility with Hive, only first character is considered
+  and remaining are ignored. (:issue:`3891`)
+* Improve performance of ``INSERT`` queries writing to bucketed tables, when some buckets to not receive
+  any data. (:issue:`1375`)
+* Improve performance of queries reading Parquet data with predicates on ``timestamp`` columns. (:issue:`4104`)
+* Improve performance for join queries over partitioned tables. (:issue:`4156`)
+* Add add support for ``null_format`` table property for tables using TEXTFILE storage format (:issue:`4056`)
+* Add add support for ``null_format`` table property for tables using RCTEXT and SEQUENCEFILE
+  storage formats (:issue:`4143`)
+* Add optimized Parquet writer. The new writer is disabled by default, and can be enabled with the
+  ``parquet_optimized_writer_enabled`` session property or the ``hive.parquet.optimized-writer.enabled`` configuration
+  property. (:issue:`3400`)
+
+PostgreSQL Connector Changes
+----------------------------
+
+* Improve performance of aggregation queries, by computing aggregations within PostgreSQL database.
+  Currently, following aggregate functions are eligible for pushdown: ``count``,  ``min``, ``max``, ``sum``
+  and ``avg``. (:issue:`3881`, :issue:`18`)
```

Comment:
I don't think we need to mention issue 18, as that one is generic and is linked to from the PostgreSQL PR.